lib/puppet-lint/plugins/check_parameter_documentation.rb

```diff
@@ -28,27 +26,30 @@ def check
       next if is_private
```

Comment:
Perhaps checks still make sense on private classes - if it documents params that don't exist it should warn IMHO. Same for duplicate params but it shouldn't warn about undocumented params.

Reply:
Very good point.  I'll look into that.

Reply:
Done